core/src/main/java/com/novoda/noplayer/mediaplayer/AndroidMediaPlayerAudioTrackSelector.java

```diff
@@ -6,7 +6,7 @@
 
 import java.util.ArrayList;
 
-class AndroidMediaPlayerTrackSelector {
+class AndroidMediaPlayerAudioTrackSelector {
```

Comment:
Renamed to match the structure of `ExoPlayer`, we might need to introduce the same level of abstraction dependending on the mockability of the objects here when testing.